front/app/modules/commercial/smart_groups/components/UserFilterConditions/PredicateSelector.tsx

```diff
@@ -69,12 +69,13 @@ const PredicateSelector = memo(
         predicate_not_is_verified: messages.predicate_not_is_verified,
         predicate_not_posted_in: messages.predicate_not_posted_input,
         predicate_not_volunteered_in: messages.predicate_not_volunteered_in,
-        predicate_not_voted_comment_in: messages.predicate_not_voted_comment_in,
-        predicate_not_voted_idea_in: messages.predicate_not_voted_input_in,
+        predicate_not_reacted_comment_in:
+          messages.predicate_not_voted_comment_in,
+        predicate_not_reacted_idea_in: messages.predicate_not_voted_input_in,
         predicate_posted_in: messages.predicate_posted_input,
         predicate_volunteered_in: messages.predicate_volunteered_in,
-        predicate_voted_comment_in: messages.predicate_voted_comment_in,
-        predicate_voted_idea_in: messages.predicate_voted_input_in,
+        predicate_reacted_comment_in: messages.predicate_voted_comment_in,
```

Comment:
These messages currently still contain the word 'vote': both the attribute names here (`messages.predicate_voted_comment_in`) and the messages themselves (see `app/modules/commercial/smart_groups/components/UserFilterConditions/messages.ts`)

Reply:
Yes message names and copy should be untouched in this release and will be changed in a later release